src/suricata.c

```diff
@@ -233,6 +233,10 @@ int g_disable_randomness = 1;
   * comparing flows */
 uint16_t g_vlan_mask = 0xffff;
 
+/* flag to disable hashing almost globally, to be similar to disabling nss
+ * support */
+bool g_disable_hashing = false;
```

Comment:
I feel the disable hashing logic should be in its own commit, not as part of a transform update.